### PR TITLE
Bug fixes and Docker work

### PR DESCRIPTION
**Pull request by vincent-zurczak**
I know, the PR's title is very explicit.

### CODE REVIEW

File: core/roboconf-target-docker/src/main/java/net/roboconf/target/docker/internal/DockerHandler.java [hunk 1]
```diff
@@ -117,7 +117,7 @@ else if(( image = DockerUtils.findImageByTag( imageId, images )) != null )
 			InputStream response = null;
 			File dockerfile = null;
 			try {
-				dockerfile = new DockerfileGenerator(pack, targetProperties.get(AGENT_JRE_AND_PACKAGES)).generateDockerfile();
+				dockerfile = new DockerFileGenerator(pack, targetProperties.get(AGENT_JRE_AND_PACKAGES)).generateDockerfile();
```
Comment (gibello) [inline core/roboconf-target-docker/src/main/java/net/roboconf/target/docker/internal/DockerHandler.java]: "DockerfileGenerator" was intentional (what we generate is called a "Dockerfile", not a "docker file").

**Comment by bourretp** [comment]
:bangbang:  Mmmh, CI test results let me think there is a little something missing, right?

**Comment by vincent-zurczak** [comment]
Yes, I'm going to push new modifications.

**Comment by vincent-zurczak** [comment]
Before merging, new manual tests will be necessary on Docker.
I had an issue with my bandwith... Concurrent **sudo apt-get update** almost killed my box in the afternoon.

**Comment by vincent-zurczak** [comment]
Docker support was tested by hand on the LAMP example.

Deploying a single root instance in Docker (and generating the image on the fly) works.
Deploying all the instances (4 in the LAMP application) and still generating the image on the fly works too. No conflict occurred during the generation.

**Comment by bourretp** [comment]
All clear!

**Pull request merged**